scripts/hurl/rs/hrl

```diff
@@ -114,6 +119,21 @@ if [ -z "$secret" ]; then
     exit 1
 fi
 
+
+if [ "$allow_outbound" = false ]; then
+  file_path=$(dirname "$fpath")
+  substitutedFilePathPrefix="$CDCTI_HOME/substituted_receivers"
+#  Make temporary directory to store scrambled file
+  mkdir -p "$substitutedFilePathPrefix/$file_path" && touch -f "$substitutedFilePathPrefix/$fpath"
+#  Grab MSH Header receiver fields
+  receiverFieldsToSubstitute=$(head -n1 "$CDCTI_HOME/$fpath" | cut -d "|" -f 5,6)
```

Comment:
Instead of replacing MSH-5/6, we can replace the whole MSH segment. That will also handle some other edge cases. For orders, we can replace it with: `MSH|^~\&|Sender Application^sender.test.com^DNS|Sender Facility^0.0.0.0.0.0.0.0^ISO|Receiver Application^0.0.0.0.0.0.0.0^ISO|Receiver Facility^simulated-lab-id^DNS|20230101010000-0000||ORM^O01^ORM_O01|111111|T|2.5.1||||||||||` and for results:  `MSH|^~\&|Sender Application^sender.test.com^DNS|Sender Facility^0.0.0.0.0.0.0.0^ISO|Receiver Application^0.0.0.0.0.0.0.0^ISO|Receiver Facility^simulated-hospital-id^DNS|20230101010000-0000||ORU^R01^ORU_R01|111111|T|2.5.1||||||||||`

Reply:
Implemented suggestions as discussed!